.github/workflows/CI.yaml

```
@@ -156,8 +156,8 @@ jobs:
     - name: Set GOPATH
       run: |
         # temporary fix for https://github.com/actions/setup-go/issues/14
-        echo "##[set-env name=GOPATH;]$(dirname $GITHUB_WORKSPACE)"
-        echo "##[add-path]$(dirname $GITHUB_WORKSPACE)/bin"
+        echo "GOPATH=$(dirname $GITHUB_WORKSPACE)" >> $GITHUB_ENV
+        echo "PATH=$(dirname $GITHUB_WORKSPACE)/bin:${PATH}" >> $GITHUB_PATH
```

```suggestion
        echo "$(dirname $GITHUB_WORKSPACE)/bin" >> $GITHUB_PATH
```
[Docs](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#adding-a-system-path) say echoing to that file prepends what's written to it.